homeassistant/components/snips/__init__.py

```diff
@@ -159,7 +167,10 @@ async def message_received(msg):
         except intent.IntentError:
             _LOGGER.exception("Error while handling intent: %s", intent_type)
 
-    await mqtt.async_subscribe(hass, INTENT_TOPIC, message_received)
+    try:
+        await mqtt.async_subscribe(hass, INTENT_TOPIC, message_received)
+    except HomeAssistantError:
+        _LOGGER.warning("MQTT has not been setup yet!'")
```

Comment:
OK, then it should be retried since MQTT starts up late, or else this integration simply won't work -- it will just log an error that the user will likely not see.  Please consult with core devs.

Reply:
Right, sorry for misunderstanding. I have created a function that allows to await the MQTT client setup. This will only throw an error if the MQTT config entry is not set up yet.